src/mobile/components/follow_button/view.coffee

```diff
@@ -52,6 +52,6 @@ module.exports = class FollowButtonView extends Backbone.View
         setTimeout (=> @$el.removeClass 'is-clicked'), 1500
     else
       analyticsHooks.trigger 'follow:login'
-      location.href = "/log_in?redirect-to=#{window.location}"
+      location.href = "/sign_up?action=follow&objectId=#{@followId}&kind=artist&redirect-to=#{window.location}"
```

Comment:
May want to go ahead and change the analytic trigger to `follow:signup`. Not sure if we have something listening for this specifically, but it should be pretty straightforward to add. It would go [here](https://github.com/artsy/force/blob/master/src/mobile/analytics/following.js).

Reply:
Not sure if that analytics event is event still relevant...